src/gasPriceOracle/adapters/linea-viem.ts

```diff
@@ -1,17 +1,23 @@
-import { PublicClient } from "viem";
+import { Address, PublicClient } from "viem";
 import { estimateGas } from "viem/linea";
 import { DEFAULT_SIMULATED_RELAYER_ADDRESS as account } from "../../constants";
 import { InternalGasPriceEstimate } from "../types";
+import { PopulatedTransaction } from "ethers";
 
-export async function eip1559(provider: PublicClient, _chainId?: number): Promise<InternalGasPriceEstimate> {
+export async function eip1559(
+  provider: PublicClient,
+  _chainId: number,
+  baseFeeMultiplier: number,
+  _unsignedTx?: PopulatedTransaction
+): Promise<InternalGasPriceEstimate> {
   const { baseFeePerGas, priorityFeePerGas } = await estimateGas(provider, {
-    account,
-    to: account,
-    value: BigInt(1),
+    account: (_unsignedTx?.from as Address) ?? account,
```

Comment:
Also @pxrl dont you find it weird we don't need to pass the transactions calldata here? If so then why does the unsigned tx even matter for priority fee pricing?